README.md

```diff
@@ -33,7 +33,7 @@ Our chat channel is on gitter here: https://gitter.im/fossasia/susi_server
 * **Step 5:** Build locally by running : ```$ npm run build ```
 * **Step 6:** To deploy at a url use : ```$ npm run deploy ```
 
-### Running on [Surge](./docs/INSTALLATION_SURGE.md)
+### Running on [Surge](https://github.com/fossasia/chat.susi.ai/blob/master/docs/INSTALLATION_SURGE_ENGLISH.md)
```

Comment:
I don't think having a link here is required at all. The README itself has all the instructions for working with Sure.